marketplacetests/tests/test_marketplace_search_and_install_app.py

```diff
@@ -25,6 +25,12 @@ def test_search_and_install_app(self):
         marketplace.launch()
 
         self.app_name = marketplace.popular_apps[0].name
+
+        # Remove the app if already installed
+        if self.apps.is_app_installed(self.app_name):
+            self.apps.uninstall(self.app_name)
```

Comment:
I threw a `print` statement in here, and it doesn't seem like this is ever called, so maybe there is something wrong with `is_app_installed`.